java/src/main/java/com/codurance/training/tasks/ViewTaskImpl.java

```diff
@@ -0,0 +1,67 @@
+package com.codurance.training.tasks;
+
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+public class ViewTaskImpl implements ViewTask{
+    @Override
+    public void viewByToday(Map<String, List<Task>> tasks) {
+        Date today = new Date();
+
+        for (Map.Entry<String, List<Task>> project : tasks.entrySet()) {
```

Comment:
in this use case instead of implementing all methods in class, it better to create different classes as welll..
bcz if we want to change any of the logic in future than then we need to modify the viewTaskImpl class...this could violate the single responsibility principle